ssz/sedes/list.py

```diff
@@ -142,20 +153,18 @@ def _deserialize_stream(self, stream: IO[bytes]) -> Iterable[TDeserialized]:
     # Tree hashing
     #
     def hash_tree_root(self, value: Iterable[TSerializable]) -> bytes:
```

Comment:
pending on https://github.com/ethereum/eth2.0-specs/pull/1283

Reply:
I was just thinking about input validation, to prevent people from accidentally hashing inputs that are too big. Don't think it's super necessary though, we can always add this later.